history/universe/marathon.json.mustache

```diff
@@ -57,7 +57,7 @@
     "healthChecks": [
         {
             "protocol": "COMMAND",
-            "command": { "value": "if [ -n \"$(tail -n500 $MESOS_SANDBOX/stderr | grep ERROR )\" ]; then echo 'Spark History Server logs contain errors'; exit 1; else exit 0; fi" },
+            "command": { "value": "echo \"healthcheck: running curl -s --fail http://$LIBPROCESS_IP:$PORT0/api/v1/version\ > /dev/null"; curl -s --fail \"$LIBPROCESS_IP:$PORT0/api/v1/version\" > /dev/null" },
```

Comment:
The quoting here looks wrong:
```
echo \"healthcheck: running curl -s --fail http://$LIBPROCESS_IP:$PORT0/api/v1/version\ > /dev/null"
```
If you escape the leading quote, then the trailing one should be escaped too. Yet you escape a space.
Also echoing this into `/dev/null` isn't very useful :-)
Also, `-o /dev/null` is slightly less awkward in cases like this than the redirection.